internal/store/postgres/flow_repository.go

```diff
@@ -126,3 +128,40 @@ func (s *FlowRepository) Delete(ctx context.Context, id uuid.UUID) error {
 		return fmt.Errorf("no entry to delete")
 	})
 }
+
+func (s *FlowRepository) DeleteExpiredFlows(ctx context.Context, logger log.Logger) error {
+	expiryTime := time.Now().UTC().AddDate(0, 0, -7)
```

Comment:
I think we should use a `Now` function to get the current time instead of `time.Now()` else we won't be able to mock the time to write unit tests. Check the session service for reference.